data/payloads/70632d81-bb39-40dd-bfba-cfd1f8196eb6.yml

```diff
@@ -0,0 +1,69 @@
+---
+
+id: 70632d81-bb39-40dd-bfba-cfd1f8196eb6
+name: Stockpile Payloads
+payloads:
+  - name: Akagi64.exe
+    description: UACME compiled binary
+    id: dcb15ae4-52fb-4aed-be23-aae69b84f5ce
+  - name: bookmark.scpt
+    description: Unknown
+    id: c354cac4-ac7c-4512-8a52-5faf880883bf
+  - name: Bypass-UAC.ps1
+    description: Invokes bypass UAC from empire scripts
+    id: 7700675a-c8f1-4634-b006-3486014535f4
+  - name: debugger.dll
+    description: DLL versions of the minidump.go binary for reflective injection
+    id: feaf09c6-6337-49dd-b301-7cf015cc7cd4
+  - name: debugger.exe
+    description: Modified cmd.exe with a Bit-flipped to maintain signature and change hash
+    id: 33b38fbf-5897-4021-9f27-9ae3f3f24c12
+  - name: Emulate-Administrator-Tasks.ps1
+    description: Script to randomly perform common administrator tasks
+    id: af7f232a-6636-46f9-a10b-148b55c3277b
+  - name: HostingCLRx64.dll
+    description: Hosting DLL for a windows C# CLR (for use with execute-assembly)
+    id: c7841398-8728-494b-9b0c-9fe5fe2524de
+  - name: Invoke-MemeKatz.ps1
+    description: Random sets your desktop to a meme cat
+    id: b2dcf04d-517a-4eca-95f2-2c41bc783125
+  - name: invoke-mimi.ps1.xored
+    description: Invoke katz script
+    xored: True
+    id: c1e0dda5-1f10-4895-a963-261404bae5f8
+  - name: Invoke-ReflectivePEInjection.ps1.xored
+    description: Invoke reflective PE injection script
+    xored: True
+    id: 37e61603-fb0a-48c4-ae9b-f6542c6b1221
+  - name: minidump.exe
+    description: Compiled minidump.go binary for dumping LSASS memory
+    id: 61174873-e8f8-486d-9f9f-12884370dde7
+  - name: powerview.ps1.xored
+    description: Powerview script from empire
+    xored: True
+    id: 14986e59-601d-4c52-9ec1-480090ab6550
+  - name: ragdoll.py
+    description: HTML-based implant
+    id: 2e1834cb-a599-44a3-b5c8-f58df4ab51e6
+  - name: reflect.ps1
+    description: PE Reflect script
+    id: 89102904-1a41-443d-b2cd-1faf2529ae20
+  - name: scanner.py
+    description: Python port scanner
+    id: 4318470d-715b-4355-aa1c-11c489d4c10e
+  - name: totallylegit.exe
+    description: Compiled minidump.go binary for dumping LSASS memory
+    id: 98c0a4cf-6fc3-446e-9857-fe396a9eafee
+  - name: wifi.ps1
+    description: Wifi manipulation script
+    id: 28f9bf43-4f14-4965-9bd9-b70fd6993d8e
+  - name: wifi.sh
+    description: Wifi manipulation script
+    id: 9f639067-370a-40ba-b7ac-6f1c15d5a158
+special_payloads:
+  - name: minidump.go
```

Comment:
will special payloads reference the function to call..?

Reply:
I'm floating back and forth on this.  We could specify the module & function. These two haven't been implemented yet so they're not `special_payloads` yet

Reply:
makes sense. I'd lean toward putting it in (and obviously validating the functions on load) but you'll have to toy with that.